tests/nightly/estimator/test_estimator_cnn_cpu.py

```diff
@@ -0,0 +1,106 @@
+    '''
+    factor = (kernel_size + 1) // 2
+    if kernel_size % 2 == 1:
+        center = factor - 1
+    else:
+        center = factor - 0.5
+    og = np.ogrid[:kernel_size, :kernel_size]
+    filt = (1 - abs(og[0] - center) / factor) * (1 - abs(og[1] - center) / factor)
+    weight = np.zeros((in_channels, out_channels, kernel_size, kernel_size), dtype='float32')
+    weight[range(in_channels), range(out_channels), :, :] = filt
+    return nd.array(weight)
+
+def FCN(num_classes=21, ctx=None):
+    '''
+    FCN model for semantic segmentation
+    '''
+    pretrained_net = vision.resnet18_v2(pretrained=True, ctx=ctx)
+
+    net = gluon.nn.HybridSequential()
+    for layer in pretrained_net.features[:-2]:
+        net.add(layer)
+
+    net.add(gluon.nn.Conv2D(num_classes, kernel_size=1),
+            gluon.nn.Conv2DTranspose(num_classes, kernel_size=64, padding=16, strides=32))
+    return net
+
+def test_estimator():
+    '''
+    Test estimator by doing one pass over each model with synthetic data
+    '''
+    models = ['resnet18_v1',
+              'alexnet',
+              'FCN'
+              ]
+    context = mx.cpu()
+    for model_name in models:
+        batch_size = 1
+        num_epochs = 1
+        lr = 0.001
+        # Get model
+        if model_name is 'FCN':
+            num_classes = 21
+            net = FCN(num_classes=num_classes, ctx=context)
+            dataset = gluon.data.dataset.ArrayDataset(mx.nd.random.uniform(shape=(batch_size, 3, 320, 480)),
+                                                      mx.nd.zeros(shape=(batch_size, 320, 480)))
+            loss = gluon.loss.SoftmaxCrossEntropyLoss(axis=1)
+            net[-1].initialize(init.Constant(bilinear_kernel(num_classes, num_classes, 64)), ctx=context)
+            net[-2].initialize(init=init.Xavier(), ctx=context)
+        else:
+            net = vision.get_model(model_name, classes=10)
+            dataset = gluon.data.dataset.ArrayDataset(mx.nd.random.uniform(shape=(batch_size, 1, 224, 224)),
+                                                      mx.nd.zeros(batch_size))
+            loss = gluon.loss.SoftmaxCrossEntropyLoss()
+            net.initialize(mx.init.MSRAPrelu(), ctx=context)
+
+        train_data = gluon.data.DataLoader(dataset, batch_size=batch_size)
+        # Define loss and evaluation metrics
+        acc = mx.metric.Accuracy()
+        # Hybridize and initialize net
+        net.hybridize()
+        # Define trainer
+        trainer = gluon.Trainer(net.collect_params(), 'sgd', {'learning_rate': lr})
+        # Define estimator
+        est = estimator.Estimator(net=net,
+                                  loss=loss,
+                                  metrics=acc,
+                                  trainers=trainer,
+                                  context=context)
+        # Call fit() to begin training
+        logging_handler = event_handler.LoggingHandler(est, model_name+'_log', model_name+'_log')
```

Comment:
LoggingHandler Class constructor says `def __init__(self, estimator, file_name=None, file_location=None, ):`, is `file_location` name correct?(e.g.: alexnet_log/alexnet_log). I think you have to provide `file_location` as a directory path or you can leave it empty since code takes care of it. Thanks!

Reply:
Thanks for pointing it out. Removing logging handler from integration tests for now as its undergoing changes. It will be covered in unit tests.